cfgov/ask_cfpb/models/django.py

```diff
@@ -151,7 +151,9 @@ def top_tags_es(self):
 
     @cached_property
     def facet_map(self):
-        answers = self.answer_set.order_by('-pk').select_related()
+        raw_answers = self.answer_set.order_by('-pk').select_related()
+        answers = [
+            answer for answer in raw_answers if answer.english_page.live]
```

Comment:
Would the result be the same if you filtered by 'live' at query-time?

Like  `self.answer_set.filter(live=True).order_by('-pk').select_related()`

Reply:
the live value isn't available in an answer query, because it lives on the english or spanish page. answer.english_page is a property that can be harvested after the query.

Reply:
It's not efficient, but this operation is done during elasticsearch indexing, not during live requests.